cmd/object-handlers.go

```diff
@@ -689,6 +689,197 @@ func (api objectAPIHandlers) getObjectHandler(ctx context.Context, objectAPI Obj
 	})
 }
 
+// GetObjectAttributes ...
+// https://docs.aws.amazon.com/AmazonS3/latest/API/API_GetObjectAttributes.html#
+func (api objectAPIHandlers) getObjectAttributesHandler(ctx context.Context, objectAPI ObjectLayer, bucket, object string, w http.ResponseWriter, r *http.Request) {
+	if crypto.S3.IsRequested(r.Header) || crypto.S3KMS.IsRequested(r.Header) { // If SSE-S3 or SSE-KMS present -> AWS fails with undefined error
```

Comment:
I am pretty sure this is not how we should behave. I'd expect we should support these. These are fundamental features - and the error is probably just because it wasn't configured on AWS.

Reply:
It seemed to behave similarly to HeadObject so I added these in, but I'll remove them.